man/rmd/indexed-error.Rmd

```diff
@@ -0,0 +1,121 @@
+The `purrr_error_indexed` class is thrown by [map()], [map2()], [pmap()], and friends.
+It wraps errors thrown by applying `.f` to individual elements giving additional information about the location of the error.
```

Comment:
I first read this as "the error is wrapped by applying `.f`".

Something along these lines might be clearer:

"When an error is thrown during the processing of an individual element by the mapped function `.f`, the error gets chained by `map()` and friends to a `purrr_error_indexed` that provides additional information about the location of the error.